wikimedia/wikipedia-ios
# Fix link colors in Talk Page `UITextView`s

**Phabricator:** https://phabricator.wikimedia.org/T323383

### Notes
This PR fixes an issue where the text view's on talk pages weren't rendering link colors correctly. It appears the preferred link color for `UITextView`s should be set via the `linkTextAttributes` property and not manually attributing the range of the link in the attributed string as `linkColor:` attempts to. I recall we were talking about this recently – apologies, couldn't find a related Phab task.

### Test Steps
1. Open Talk Page for user or article that has links in comments/coffee roll
2. Confirm links render with expected color (`theme.colors.link`)
3. Change theme, confirm links render with excepted color
